game/resource/addon_english.txt

```diff
@@ -228,6 +228,8 @@
         "Goal_6_Opening_1" "Watch Centaur Warrunner's (Strength) skill showcase."
         "Goal_6_Opening_2" "Watch Phantom Assassin's (Agility) skill showcase."
         "Goal_6_Opening_3" "Watch Invoker's (Intelligence) skill showcase."
+        "Goal_6_Closing_1"  "Destroy the enemy towers defending the Ancient"
```

Comment:
Period to be consistent?

Reply:
It shows "Destroy the enemy towers defending the Ancient: 0/2", a period would look weird here.